### PR TITLE
AO3-4441 Automated tests for importing and rearranging prompt memes

### PR DESCRIPTION
**Pull request by cesy**
https://otwarchive.atlassian.net/browse/AO3-4441

Improve the automated tests for importing, which currently don't cover several key features. Adds tests for getting a claim email when someone imports your work, responding to that email to claim your work, responding to orphan, and responding to delete and ask not to be contacted again.

It appears that responding to claim is not working entirely smoothly, needs further investigation later. In rough testing on Staging, I managed to get an error 500 by clicking the claim link, then logging out. The test fails when you try to claim, with error 500 about login not existing.

Also rearrange the prompt meme tests that keep failing on Travis so they're of more equal size and logically arranged. Plenty more to optimise there, but keeping them sensibly sized will reduce failures.

### CODE REVIEW

File: features/importing/archivist.feature [hunk 1]
```diff
@@ -110,13 +110,63 @@ Feature: Archivist bulk imports
...
 
   Scenario: Claim a work and create a new account in response to an invite
-  # TODO
+
+    Given I have an archivist "elynross"
+      And the default ratings exist
+      And account creation is enabled
+    When I am logged in as "elynross"
+      And I go to the import page
+      And I import the work "http://cesy.dreamwidth.org/154770.html" by "randomtestname" with email "otwstephanie@thepotionsmaster.net"
```
Comment (emma-burrows) [inline features/importing/archivist.feature]: I think this is probably a good opportunity to remove the personal email addresses of former volunteers, while we're at it!

Comment (ariana-paris) [inline features/importing/archivist.feature]: Probably worth taking this opportunity to remove the personal email addresses of former volunteers, while we're at it!

File: features/importing/archivist.feature [hunk 2]
```diff
@@ -135,3 +185,12 @@ Feature: Archivist bulk imports
     When I check the 1st checkbox with id matching "importing_for_others"
     And I press "Import"
     Then I should see "We have notified the author(s) you imported works for. If any were missed, you can also add co-authors manually."
+
+  Scenario: Update the URL on an imported work with a redirect
+
+  # Given I have an Open Doors committee member "Ariana"
```
Comment (ariana-paris) [inline features/importing/archivist.feature]: O.O
:D
However, if this pull request is ready to go, can we not have commented out code in it? It has a habit of lingering and it's generally good practice to reduce comment-induced noise.

File: features/prompt_memes_a/challenge_promptmeme_setup.feature [hunk 3]
```diff
@@ -228,7 +228,6 @@ Feature: Prompt Meme Challenge
   # TODO: We need to check the display for fandomless memes
```
Comment (ariana-paris) [inline features/prompt_memes_a/challenge_promptmeme_setup.feature]: While I'm ranting about comments, TODOs are something else that tend to linger and add nothing to the repository code. For example, the ones in these challenge tests are all from 2011. (so not part of the pull request, but an illustration of why you should pretty much never commit code with TODOs in it because in general ... you won't!)

**Comment by ariana-paris** [comment]
Just as a general note, pull requests should be about a single thing. This one is about two things: import tests (which I can review since I work on the importing code), and rearranging challenge tests (which I can't really express an opinion on beyond "yup, they've been rearranged" as I'm not familiar with the challenge code). They would have been easier to review as separate entities (for future reference).

But aside from a couple of comments, it looks good to go.

File: features/importing/archivist.feature (hunk 2 above)
Comment (cesy) [inline features/importing/archivist.feature]: Aha, that's because there were bits of OD tests in the other PR - I'll clean this out for now, and then can expand it once both are merged. Thanks for highlighting.

File: features/prompt_memes_a/challenge_promptmeme_setup.feature (hunk 3 above)
Comment (cesy) [inline features/prompt_memes_a/challenge_promptmeme_setup.feature]: Yeah, it's a good example. I'm hoping to go back and fix some of those soon.

File: features/importing/archivist.feature (hunk 1 above)
Comment (cesy) [inline features/importing/archivist.feature]: Done.

**Comment by cesy** [comment]
Okay, fixed all those, I think. For the rearranged stuff, it's just checking I didn't accidentally delete or add anything in the process - I'll do a separate PR once I start actually improving them.

**Comment by cesy** [comment]
Travis failures just need a re-run.
